src/expressions/operations.py

```diff
@@ -59,10 +81,16 @@ def __repr__(self):
     
     def expand(self):
         return Negative(self.exp.expand())
+    
+    def contains(self, var):
```

Comment:
Because we are only looking at one thing instead of:
```
if self.exp.contains(var):
            return True
        else:
            return False
```
I would do the following:
`return self.exp.contains(var)`